mod/deckorate_attribution/assets/script/clipboard.js.coffee

```diff
@@ -1,9 +1,10 @@
 $ ->
   $("body").on "click", ".copy-button", ->
-    clipboardContent = $("#clipboard").text()
-    
-    navigator.clipboard.writeText(clipboardContent)
+    content = $(this).closest(".tab-pane.active").find("._clipboard").html()
```

Comment:
Is this change from `text()` to `html()` what fixed the hyperlinks? Or was the Blob thing also necessary?

Do you get different behavior copying from the html and the rich text tabs? What's different?